app/models/repository.rb

```diff
@@ -6,4 +6,12 @@ class Repository < ApplicationRecord
   def to_s
     link
   end
+
+  scope :languages, -> { select(:language).distinct.where.not(language: [nil, '']) }
+  scope :by_repository_name_like, lambda { |repository_name|
+    if repository_name.present?
```

Comment:
what is the need of this check?

Reply:
Some repositories has blank values (specially repositories from the seeds), it didn't make sense to me to show a blank language on the languages filter so i did this check

Reply:
the argument is required, if the dev is passing an empty string it's the dev's fault